docs/user/security/tutorials/how-to-secure-access-to-kibana.asciidoc

```diff
@@ -0,0 +1,133 @@
+
+You can create your own roles, or used any of the {ref}/built-in-roles.html[built-in roles]. Note that some of the built-in roles are intended for Elastic stack components, and should not be assigned to end users directly.
+
+One of the more useful built-in roles is `kibana_admin`. Assigning this role to your users will grant access to all if {kib}'s features. This includes the ability to manage Spaces.
+
+These built-in roles are great for getting started with the stack, and for system administrators who do not need more restrictive access. With so many features, it’s not possible for us to ship more granular roles to accommodate everyone’s needs. This is where custom roles come in. As an administrator, you have the ability to create your own roles to describe exactly the kind of access your users should have. For example, you might create a `marketing_user` role, which you then assign to all users in your marketing department. This role would grant access to all of the necessary data and features for this team to be successful, without granting them access they don’t require.
+
+
+[float]
+=== Users
+
+Once your roles are setup, the next step to securing access is to create your users, and assign them one or more roles. {kib}'s user management allows you to provision accounts for each of your users.
+
+TIP: Want Single Sign-on? {kib} supports a wide range of SSO implementations, including SAML, OIDC, LDAP/AD, and Kerberos. <<kibana-authentication, Learn more about {kib}'s SSO features>>.
+
+
+[float]
+[[tutorial-secure-kibana-dashboards-only]]
+=== Example: Create a user with access only to dashboards
+
+Let’s work through an example together. Consider a marketing analyst who wants to monitor the effectiveness of their campaigns. They should be able to see their team’s dashboards, but they should not be allowed to view or manage anything else in {kib}. All of their team’s dashboards are located in the `Marketing` space.
+
+[float]
+==== Create a space
+
+Create a `Marketing` space for your marketing analysts to use.
+
+. Open the main menu, and select **Stack Management**.
+. Under **{kib}**, select **Spaces**
+. Click **Create a space**.
+. Give this space a unique name. For example: `Marketing`.
+. Click **Create space**.
+
+If you’ve followed the example above, you should end up with a space that looks like this:
+
+[role="screenshot"]
+image::user/security/images/tutorial-secure-access-example-1-space.png[Create space UI]
+
+
+[float]
+==== Create a role
+
+Create a role that describes the privileges you want to grant.
+
+To effectively use dashboards, the marketing analyst will need:
+
+* Access to **read** the data that powers the dashboards
+* Access to **read** the dashboards within the `Marketing` space.
+
+. Open the main menu, and select **Stack Management**.
```

Comment:
This should not be indented under the "Access to read the dashboards..."  bullet point.  Instead, it should be step 1.